### PR TITLE
CIV-5433 - Add Title validation, Add Code Owners

### PR DESCRIPTION
**Pull request by chris-pateman**
### JIRA link (if applicable) ###
CIV-5433

### Change description ###
Add Title validation
Add Code Owners 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Comment by filippo-possenti-version1** [comment]
I'm assuming that there's renovate in place of dependabot.

Mind that before removing dependabot we may want to make sure renovate is in place and configured appropriately. In this case I would assume the renovate.json file needs to be added with some configuration similar to that of civil-service.

Can't approve as I don't own this repo and I'm not sure what's in it exactly but I would expect objections as in the above.

**Comment by chris-pateman** [comment]
> I'm assuming that there's renovate in place of dependabot.
> 
> Mind that before removing dependabot we may want to make sure renovate is in place and configured appropriately. In this case I would assume the renovate.json file needs to be added with some configuration similar to that of civil-service.
> 
> Can't approve as I don't own this repo and I'm not sure what's in it exactly but I would expect objections as in the above.

After our talk, I have readded dependabot back in and we can tackle this in another PR.

File: .github/workflows/main.yml
Comment (LeonardoPalmeiro) [inline .github/workflows/main.yml:19]: please add new line at the end of file.

**Review by LeonardoPalmeiro — DISMISSED** [review]

File: .github/workflows/main.yml
Comment (chris-pateman) [reply .github/workflows/main.yml:19]: done

**Review by filippo-possenti-version1 — DISMISSED** [review]

**Review by kurt-uk — APPROVED** [review]
approved but github is saying the codeowners with `infra` in them arent recognised

File: .github/workflows/main.yml
Comment (sabahirfan) [inline .github/workflows/main.yml:18]: what if we don't have a Jira for some quick task? i.e. fix a build etc.

Comment (chris-pateman) [reply .github/workflows/main.yml:18]: Everything should have a task so it can be traced back. A hack round it would be add a Prefix of CIV-9999

**Review by sabahirfan — APPROVED** [review]

**Pull request merged**
